pkg/web/apps/apps_psql_repository.go

```diff
@@ -36,3 +37,53 @@ func (a *appsPostgreSQLRepository) GetApps(c echo.Context) (*[]models.App, error
 
 	return &apps, nil
 }
+
+// GetAppsId retrieves an app by id from the database
+func (a *appsPostgreSQLRepository) GetAppsID(ID string) (*[]models.App, error) {
```

Comment:
should be renamed to GetAppbyId in my opinion as we are retrieving a single app using it's Id

Reply:
:+1: